light-zk.js/src/index.ts

```diff
@@ -11,3 +11,4 @@ export * from "./errors";
 export * from "./types";
 export * from "./transaction";
 export * from "./convertCase";
+export * as circuitlibjs from "@lightprotocol/circuit-lib.js";
```

Comment:
Is this re-export really needed?

Reply:
yes to avoid the linking issue I describe in the comment above.
- zk.js build on the linked version of circuit-lib.js
- if we import both the linked circuit-lib.js and zk.js yarn complains